playwright/src/test/java/com/microsoft/playwright/TestBrowserContextBasic.java

```diff
@@ -92,11 +97,12 @@ void shouldIsolateLocalStorageAndCookies() {
     context1.close();
     context2.close();
 
+    //Failing is 1 expected because of the old TestBase class? Should it be zero since we are only creating 2 contexts and closing both?
     assertEquals(1, browser.contexts().size());
```

Comment:
here is the other place it is failing because of the old TestBase's default browser context creation.

Reply:
Yes, you are right. Previously we were creating a context unconditionally in beforeEach and we assert that the closed contexts are removed from the contexts() list.